projects/core/src/lib/layers/symbology/raster-symbology-editor/raster-symbology-editor.component.scss

```diff
@@ -24,3 +19,29 @@ mat-slider {
         margin: 0 auto;
     }
 }
+
+#gradientType {
+    flex-wrap: wrap;
```

Comment:
![grafik](https://user-images.githubusercontent.com/4947061/202225537-068e2989-ed91-457c-acc6-980f30663cb1.png)

1. This looks bad. It would be better to show the options as one item per row.
2. Please use the Angular flex layout selectors (https://github.com/angular/flex-layout) on the components when possible.

Reply:
Each button is now displayed in its own row, and I moved the "Colorizer" text to the top as well. Also added a divider between the color pickers and colorizer settings. f2c0f44af28f79d4d5bb91812260706d2c463892
![image](https://user-images.githubusercontent.com/57327151/203062048-0f34dc13-dfa8-4efd-a000-9e3792b3870d.png)